backend/Techfesia2019/event_registrations/models.py

```diff
@@ -26,9 +26,46 @@ class Team(models.Model):
 
     create_date = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def team_members(self):
+        name_list = []
+        for i in self.teammember_set.all():
+            name_list.append(i.profile.user.first_name + ' ' + i.profile.user.last_name)
+        return ','.join(name_list)
+
+    @property
+    def leader(self):
+        return self.team_leader.user
+
+    @property
+    def members(self):
+        return self.teammember_set.filter(invitation_accepted=True)
+
+    @property
+    def invitees(self):
+        return self.teammember_set.filter(invitation_accepted=False)
+
     @property
     def member_count(self):
-        return self.teammember_set.filter(invitation_accepted=True).count()
+        return 1 + self.teammember_set.filter(invitation_accepted=True).count()
+
+    @property
+    def is_reserved(self):
+        try:
+            if self.team_leader.college.name == 'Indian Institute of Information Technology, Sri City':
+                for i in self.teammember_set.all():
```

Comment:
college_name is changed to foreign key right?
How is this working?

Reply:
Yes college is now a Foriegn Key.
This checks that none of the member of a team belongs to IIITS to be a reserved team.